slashing-protection/src/test/java/tech/pegasys/web3signer/slashingprotection/dao/SignedAttestationsDaoTest.java

```diff
@@ -280,6 +280,21 @@ public void findsNearestAttestationForTargetEpoch(final Handle handle) {
         .isEqualToComparingFieldByField(attestation(1, 7, 8, 1));
   }
 
+  @Test
+  public void insertsAttestationWithIdGreaterThanIntBoundary(final Handle handle) {
```

Comment:
nit: Wonder if it could be clearer that we're dealing with a former boundary case, rather than a current boundary case. At a glance, I would assume this is testing the actual boundary

Reply:
It's testing the old boundary. I'll change to name to make it clear the intent is to make sure we can insert ids with a value greater than the integer max value